packages/create-sitecore-jss/src/templates/nextjs/src/Layout.tsx

```diff
@@ -9,6 +9,8 @@ import {
 import { getPublicUrl } from '@sitecore-jss/sitecore-jss-nextjs/utils';
 import Navigation from 'src/Navigation';
 import Scripts from 'src/Scripts';
+// The bundle imports external (BYOC) components into the app and makes sure they are ready to be used
+import ExternalComponentBundle from 'src/byoc-imports';
```

Comment:
This may not only be _external_ (imported) components - it may include local BYOC components as well.

Reply:
The "external component" is the documentation name for BYOC. It can also be "External React components".
I think the idea is BYOC components are "external" as opposed to Sitecore components being "internal"